python/src/aac/cli/execute.py

```diff
@@ -0,0 +1,72 @@
+"""The command line processor for aac."""
+import argparse
+import inspect
+import itertools
+import sys
+from typing import Callable
+
+from pluggy import PluginManager
+
+from aac.plugins.plugin_manager import get_plugin_manager
+
+
+def run_cli():
+    """
+    Run the specified AaC command.
+
+    This method parses the command line and performs the requested user command or outputs usage.
+    """
+    plugin_manager = get_plugin_manager()
+
+    arg_parser, aac_plugin_commands = _setup_arg_parser(plugin_manager)
+
+    args = arg_parser.parse_args()
+
+    for command in aac_plugin_commands:
+        if args.command == command.name:
+
+            keyword_args = {}
+            args_dict = vars(args)
+
+            # Leverage inspect here to make sure that the arg names we're trying to access are
+            # sourced from the target function itself
+            parameters = inspect.signature(command.callback).parameters.keys()
+            for argument in parameters:
+                keyword_args[argument] = args_dict[argument]
+
+            result = command.callback(**keyword_args)
+            print(f"{result.name}: {result.status_code.name.lower()}\n\n" + "\n".join(result.messages))
```

Comment:
I forgot to update this when I was in this file but I think we should use this function now that we have it.

```suggestion
            print(f"{result.name}: {result.status_code.name.lower()}\n\n{result.get_messages_as_string()}")
```